images/linux/toolset-1604.json

```diff
@@ -34,6 +34,18 @@
                 "12.*",
                 "14.*"
             ]
+        },
+        {
+            "name": "go",
+            "url" : "https://raw.githubusercontent.com/akv-platform/go-hostedtoolcache/update-versions-manifest-file/versions-manifest.json",
+            "arch": "x64",
+            "platform" : "linux",
+            "versions": [
+                "1.11.*",
+                "1.12.*",
+                "1.13.*",
+                "1.14.*"
+            ]
```

Comment:
You should setup default version in all toolset pipelines

Reply:
Done.